include/sway/config.h

```diff
@@ -6,6 +6,7 @@
 #include <time.h>
 #include <wlr/types/wlr_box.h>
 #include <xkbcommon/xkbcommon.h>
+#include "../include/config.h"
```

Comment:
We don't use relative `#include` directives. Instead, `config.h` should work.

Reply:
I don't think this works because in this case `"config.h"` refers to this file, not the generated `config.h`. Changing it to `config.h` just happens to work because below, `sway/tree/root.h`  imports the `config.h` we want.